pieforproviders/pieforproviders
# Feature/84/fix xls csv

# 💅🏼 What issue does this fix?
#84 some bugs that seem browser dependent

Sometimes the `type` of an imported XLS file is an empty string, so we're checking both type and file extension

Also, sometimes the index is `undefined` when mapping/reducing the data rows(i.e. if there are 25 columns, we will have 0-24 defined and then an undefined 25th column), so I put a try-clause `?` before `.key` so it won't choke on `.key undefined method for nilClass`

## 🍂 Before You Submit


## 🧳 Steps to test
- visit `/cases/import`
- upload the test csv, xls, and xlsx files
- you should see the data no matter the file type

## 🕯 Caveats, concerns
